### ISSUE TITLE
Polls

### ISSUE THREAD

**Issue by Coca162** [TudbuT/oxibot#11]
A poll functionality allows users to input choices that can then be voted on by the public.

### Issues
The primary difficulty is getting user input back, some bots use reaction but `ctx.say()` does not return a message as that message could actually be ephemeral. The harder way would probably be reactions and they have the downside of not allowing for voting while the bot is shut down.

**Comment by abrasmag3** [comment]
> `ctx.say` does not return a message

No, but it returns a `Result<ReplyHandle>`, which can be used to get the message

**Comment by abrasmag3** [comment]
Most implementations of polling won't work when the bot is down. Do you have any ideas for one that will?

**Comment by Coca162** [comment]
> > `ctx.say` does not return a message
> 
> No, but it returns a `Result<ReplyHandle>`, which can be used to get the message

Ah I found the `.into_message()` method on ReplyHandle

**Comment by Coca162** [comment]
> Most implementations of polling won't work when the bot is down. Do you have any ideas for one that will?

I think a reactions implementation will be the best, with in the future optional time limits that will can tracked by a tokio thread. This way we simply get the votes at the end of the limit and not use events
